### PR TITLE
PLAT-133997: Add creating agate app example in the readme.md

### PR DESCRIPTION
**Pull request by seunghoh**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
Need to explain how to create Agate App

### Resolution
Add example in the readme.md

Enact-DCO-1.0-Signed-off-by: Seungho Park <seunghoh.park@lge.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ A theme is a collection of components all styled the same way, with the same gen
 
 This theme composes a few dozen components, each of which is styled to represent several different "skins". However, the experimental part is that these "skins" are not simply color swapping, but include shapes, measurements, layouts, and arrangements, making them less like the Enact definition of a skin and more like "skin+". This approach is the middle-ground between a skin and a theme. All of the components here share a common set of behavioral and interaction features while looking completely different.
 
+## Create Agate App
```
Comment (MikyungKim) [inline README.md:15]: I'll go like... 
```suggestion
## Let's create your own Agate App
```

After ## Let me see it!

**Review by MikyungKim — APPROVED** [review]
:lgtm:

File: README.md [hunk 2]
```diff
@@ -12,6 +12,16 @@ A theme is a collection of components all styled the same way, with the same gen
 
 This theme composes a few dozen components, each of which is styled to represent several different "skins". However, the experimental part is that these "skins" are not simply color swapping, but include shapes, measurements, layouts, and arrangements, making them less like the Enact definition of a skin and more like "skin+". This approach is the middle-ground between a skin and a theme. All of the components here share a common set of behavioral and interaction features while looking completely different.
 
+## Let's create your own Agate App
+The Enact CLI itself can install the agate template via NPM or git URI.
```
Comment (0x64) [inline README.md:16]: Please add a blank line here.

File: README.md [hunk 3]
```diff
@@ -12,6 +12,16 @@ A theme is a collection of components all styled the same way, with the same gen
...
+```
+enact template install @enact/template-agate
+```
+Once installed, you can specify the `agate` template during new application creation:
```
Comment (0x64) [inline README.md:20]: Please add blank lines above and below this line.

**Review by 0x64 — COMMENTED** [review]
@seunghoh Could you please add blank lines? It will not affect the rendered markdown result but it could be helpful for editors (or some users) to see a MD file as text.

**Comment by seunghoh** [comment]
@0x64 Updated! Thanks

**Pull request merged**